app.js

```diff
@@ -17,7 +17,7 @@ app.use(logger("dev"));
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
-app.use(express.static(path.join(__dirname, "public")));
+app.use('/static', express.static('public'))
```

Comment:
You're serving statics files over the ```/static``` route, your but the HTML does not use the  ```/static``` route to fetch the files....
This change will mean that all statics files should served over ```/static```

Revert to the root route